### PR TITLE
Use EOT for ASB-MSMQ bridge sample

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: samples/azure/azure-service-bus-msmq-bridge/ASB_7/AsbEndpoint/Program.cs [hunk 1]
```diff
@@ -57,8 +68,9 @@ static async Task Main()
                 break;
             }
 
-            await endpointInstance.Send(new MyCommand { Property = "command from ASB endpoint" }).ConfigureAwait(false);
-            Console.WriteLine("\nCommand sent");
+            var prop = new string(Enumerable.Range(0, 3).Select(i => letters[random.Next(letters.Length)]).ToArray());
+            await endpointInstance.Send(new MyCommand { Property = prop }).ConfigureAwait(false);
+            Console.WriteLine($"\nCommand {prop} sent");
```
Comment (SeanFeldman) [inline samples/azure/azure-service-bus-msmq-bridge/ASB_7/AsbEndpoint/Program.cs]: Did you mean
> `Command with value={prop} sent`

Command is always `MyCommand` 🙂

File: samples/azure/azure-service-bus-msmq-bridge/ASB_7/Bridge/Program.cs [hunk 2]
```diff
@@ -24,23 +24,32 @@ static async Task Main(string[] args)
...
             });
 
         bridgeConfiguration.AutoCreateQueues();
         bridgeConfiguration.UseSubscriptionPersistece<InMemoryPersistence>((configuration, persistence) => { });
+        bridgeConfiguration.TypeGenerator.RegisterKnownType(typeof(OtherEvent));
 
         #endregion
 
+        #region resubscriber
+        var resubscriber = await Resubscriber<MsmqTransport>.Create("Bridge-MSMQ", TimeSpan.FromSeconds(10), t => { });
```
Comment (SeanFeldman) [inline samples/azure/azure-service-bus-msmq-bridge/ASB_7/Bridge/Program.cs]: > TimeSpan.FromSeconds(10), t => { }

I suspect it's a refresh interval and refresh action?
Adding parameter names would make it clear.

File: samples/azure/azure-service-bus-msmq-bridge/sample.md [hunk 3]
```diff
@@ -72,9 +76,15 @@ Bridge process is connecting between MSMQ and Azure ServiceBus transports and pr
 
 snippet: bridge-general-configuration
 
+In addition the general configuration of the transport, the `EndpointOrientedTopology` has its own requirements. The first is that all events published on the Azure Service Bus side of the bridge need to be assigned endpoints that publish them. The second is the configuration of a *resubscriber*
+
+snippet: resubscriber
+
+The resubscriber replays periodically the subscription messages coming from the MSMQ side of the bridge to ensure that the Azure Service Bus topoc notifiers have been started.
```
Comment (SeanFeldman) [inline samples/azure/azure-service-bus-msmq-bridge/sample.md]: typo in
> Azure Service Bus top**i**c notifiers

**Comment by SzymonPobiega** [comment]
@SeanFeldman it took me quite some time but I finally managed to fix the issues you pointed out

**Pull request merged**